app/pages/products/hosting-web/hosting-web.ts

```diff
@@ -72,19 +72,44 @@ export class HostingWebPage extends ProductCore {
           }
         );
     } else {
-      this.server.ssl = Object.assign({}, this.server.ssl, {status: 'deleting'});
-      this.hostingWebService.deleteSsl(this.serviceName)
-        .subscribe(
-          () => this.toast.success('La suppression de votre certificat SSL est en cours...').present(),
-          (err) => {
-            this.server.ssl = Object.assign({}, this.server.ssl, {status: 'created'});
-            this.toast.error('Une erreur est survenue lors de la suppression de votre certificat SSL : ' + JSON.parse(err._body).message).present();
-          }
-        );
+      let success = () => {
```

Comment:
Oh, I don't know why, but i test it only in the callbacks...

But do it before the alert works :)